packages/core/src/Ponder.ts

```diff
@@ -206,32 +190,20 @@ export class Ponder {
     // are triggered by changes to project files (handled in BuildService).
     this.buildService.buildSchema();
     await this.buildService.buildHandlers();
-
-    return undefined;
   }
 
   async dev() {
-    const setupError = await this.setup();
+    await this.setup();
 
     this.common.telemetry.record({
       event: "App Started",
       properties: {
         command: "ponder dev",
-        hasSetupError: !!setupError,
         logFilterCount: this.logFilters.length,
         databaseKind: this.eventStore.kind,
       },
     });
 
-    if (setupError) {
-      this.common.logger.error({
-        service: "app",
-        msg: setupError.message,
-        error: setupError,
-      });
-      return await this.kill();
-    }
```

Comment:
Nice to get rid of this, it was always a hack. We're slowly adding more granular validation for each component separately.